how-to-guides/data-versioning/scripts/Compare_model_training_runs_on_dataset_versions.py

```diff
@@ -5,7 +5,7 @@
 TRAIN_DATASET_PATH = "../datasets/tables/train.csv"
 TEST_DATASET_PATH = "../datasets/tables/test.csv"
 
-PARAMS = {
+params = {
```

Comment:
Same comment as for the earlier file: curious why this change here?

Reply:
these are the ones we are logging and can change between runs